projects/packages/wp-js-data-sync/src/class-data-sync.php

```diff
@@ -1,17 +1,165 @@
 <?php
 /**
- * Package description here
+ * This is the main file for the Data_Sync package.
+ *
+ * It's responsible for setting up the registry and the endpoints.
+ *
+ * Setting up with something like this:
+ *
+ * ```
+ *      class Widget_Status extends Data_Sync_Handler {}
+ *      class Widget_Data extends Data_Sync_Handler {}
+ *
+ *      $instance = Data_Sync::setup( 'jetpack_boost', 'jetpack-boost' );
+ *      $instance->register( 'widget_status', new Widget_Status() );
+ *      $instance->register( 'widget_data', new Widget_Data() );
+ *
+ * ```
+ *
+ * This will to create two endpoints: `/wp-json/jetpack-boost/widget_status` and `/wp-json/jetpack-boost/widget_data`
```

Comment:
I might be mixing things up, but didn't endpoints got underscores replaced with dashes? Referencing [this](https://github.com/Automattic/jetpack/blob/73f8544e1259e87a399102aef5147a06e5056740/projects/js-packages/svelte-data-sync-client/src/Endpoint.ts#L14).

Reply:
A typo!